### ISSUE TITLE
 Dev Tools Not Displaying All Elements as Expected According to Documentation

### ISSUE THREAD

**Issue by virtualabishek** [javascript-tutorial/en.javascript.info#3735]
The Dev Tools are not displaying elements as described in the documentation. According to the instructions, it should direct me to click on line 4 in hello.js. However, there is no separate line 4; the content (e.g., say(phrase)) is part of the same line, so I can't click on it.

For reference, here is the debugging page from the documentation: [JavaScript Info Debugging Page](https://javascript.info/debugging-chrome#breakpoints)

Here is an image showing on the documentation:
![image](https://github.com/user-attachments/assets/4cb06455-8bce-4e48-80c9-1f28efea427c)

For comparison, here is my own image:

![image](https://github.com/user-attachments/assets/721126b9-4ead-493b-bb5e-86dcb80a0c98)

Browser Details:

Chrome - Version 127.0.6533.89 (Official Build) (64-bit) on Windows 11 Operating System

**Comment by virtualabishek** [comment]
Yes. Thanks for the creating the issue. Still there is a same problem.  Lets hope they will fix this soon.

**Comment by iliakan** [comment]
I hope someone submits a PR to fix it.

**Comment by virtualabishek** [comment]
yes. let's hope. I dont know, why this is not quite active, although it has a great resources.

**Comment by sujayhegde23** [comment]
@iliakan @virtualabishek 
I think I may have found the cause.

From the screenshots, it looks like "hello.js" is being displayed as a single formatted line in DevTools rather than matching the formatting shown in the documentation.

One possible fix is to format  the actual javascript file .
The other way is to  click on the PRETTY PRINT (looks something like this -> { } ) icon present  below the editor pane in the  sources  panel .

screenshot for reference
<img width="957" height="982" alt="Image" src="https://github.com/user-attachments/assets/c997bef9-a391-4d09-9b4f-8523b9855bd6" />
